pytests/test_errors.py

```diff
@@ -318,34 +327,57 @@ def testFunctionListWithUnknownOption(env):
     env.expect('RG.FUNCTION', 'LIST', 'FOO').error().contains('Unknown option')
 
 @gearsTest()
-def testMalformedLibarayMetaData(env):
-    code = 'js name=foo' # no #!
-    env.expect('RG.FUNCTION', 'LOAD', code).error().contains('could not find #! syntax')
+def testMalformedLibraryMetaData(env):
+    code = 'js api_version=1.0 name=foo' # no shebang(#!)
+    env.expect('RG.FUNCTION', 'LOAD', code).error().contains('Invalid or missing prologue.')
+
+@gearsTest()
+def testMalformedLibraryMetaData2(env):
+    code = '#!js name' # no api version and no module version, invalid prologue.
+    env.expect('RG.FUNCTION', 'LOAD', code).error().contains('Invalid or missing prologue.')
+
+@gearsTest()
+def testMalformedLibraryMetaData3(env):
+    code = '#!js api_version=1.0 foo=bar' # unknown property
+    env.expect('RG.FUNCTION', 'LOAD', code).error().contains('The module name is missing from the prologue.')
```

Comment:
This error should be `The library name is missing from the prologue.`. We call the user code a RedisGears library. this matches Redis function, the code there also called library.